src/app/CreateRecording/CustomRecordingForm.tsx

```diff
@@ -101,6 +107,34 @@ export const CustomRecordingForm = (props) => {
     setRecordingName(name);
   };
 
+  const handleMaxAgeChange = (evt) => {
+    setMaxAge(Number(evt));
+  };
+
+  const handleMaxAgeUnitChange = (evt) => {
+    setMaxAgeUnits(Number(evt));
+  };
+
+  const handleMaxSizeChange = (evt) => {
+    setMaxSize(Number(evt));
+  };
+
+  const handleMaxSizeUnitChange = (evt) => {
+    setMaxSizeUnits(Number(evt));
+  };
+
+  const handleToDiskChange = (checked, evt) => {
+    setToDisk(evt.target.checked);
+  };
+
+  const setRecordingOptions = (options) => {
```

Comment:
I think adding a type annotation to `options` here would make sense and improve readability